x-pack/plugins/security_solution/public/timelines/components/side_panel/index.tsx

```diff
@@ -52,6 +53,7 @@ export const DetailsPanel = React.memo(
     isReadOnly,
   }: DetailsPanelProps) => {
     const dispatch = useDispatch();
+    const isNewFlyoutEnable = useIsExperimentalFeatureEnabled('newUserDetailsFlyout');
```

Comment:
Should we rename it to `isNewUserDetailsFlyoutEnable` because there is a new flyout in development  by investigations team as well.